fuzz/misc.cpp

```diff
@@ -48,6 +48,30 @@ void detect(std::span<const char> chardata) {
   }
 }
 
+void validate_base64(std::span<const char> chardata) {
```

Comment:
this file implements a differential fuzzer which does not make sense when all implementations are using the same fallback function.
I would rather see unit tests added.